applications/pivot/src/pivot_call.erl

```diff
@@ -148,9 +150,11 @@ init([Call, JObj]) ->
 
     ?MODULE:new_request(self(), VoiceUri, Method, BaseParams),
 
+    Counter = whapps_call:kvs_fetch('pivot_counter', 1, Call),
+
     {'ok'
      ,#state{cdr_uri=wh_json:get_value(<<"CDR-URI">>, JObj)
-             ,call=Call
+             ,call=whapps_call:kvs_update_counter('pivot_counter', Counter, Call)
```

Comment:
`Counter` should be `1`, as that parameter is the Increment to add, rather than setting the value to `Counter`.

Meaning, as you have it now, `pivot_counter` would be 1, 2, 4, 8, 16,...